docs/development/chromium-development.md

```diff
@@ -12,3 +12,64 @@
 - [Discussion Groups](http://www.chromium.org/developers/discussion-groups)
 
 See also [V8 Development](v8-development.md)
+
+# Chromium content development with electron
```

Comment:
It is usually called "Chromium content module", but in this documentation I think we should just say "Chromium", since the other parts like `net` and `v8` are also used.